src/scss/webmapviewer-bootstrap-theme.scss

```diff
@@ -11,4 +11,5 @@ $primary: $red;
   @each $key, $value in $grid-breakpoints {
     #{unquote($key)}: $value;
   }
+  test: $red;
```

Comment:
Is that an artifact from a test you did or is this meant to be here?

Reply:
oops, yeah I can also remove the part above, as with Vue3 the webpack config has been changed and SCSS files aren't compiled to JS anymore (I couldn't find an easy way to reactivate that, so instead I added the threshold in `config.js`...)